src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,261 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressWithNearbyTextFilter">
+      <p>Since Checkstyle 10.8.2</p>
+      <subsection name="Description" id="SuppressWithNearbyTextFilter_Description">
+        <p>
+          Filter <code>SuppressWithNearbyTextFilter</code> uses plain text to suppress
+          nearby audit events. The filter can suppress all checks which have Checker
+          as a parent module.
+        </p>
+        <p>
+          Rationale: Sometimes there are legitimate reasons for suppressing a check.
+          When this is a matter of the code in question and not personal preference,
+          the best place to override the policy is in the code itself. Semi-structured
```

Comment:
> the best place to override the policy is in the code itself

it is not best, it is just one of approaches that it convenient on occasion.
We do not use this model in majority of cases, as we do not like pollute code with tool comments.

Reply:
@romani Honestly, why not remove this whole rationale paragraph? I copied it from some other check. I think it just bloats the documentation and does not contribute anything now that I think.

Reply:
I think we can get rid of "Rationale", but we should extend description with a bit of the content from it. Also - check is called `SuppressWithNearbyTextFilter`, but we keep mentioning comments. @stoyanK7 are we limited to comments, or are we able to pick up any text?

Reply:
Please remove this note block

Reply:
Removed.